src/globals/theme/_index.scss

```diff
@@ -8,7 +8,7 @@
 
 @import 'functions';
 
-// TODO: `2.x` - To be removed.
+// TODO: `3.x` - To be removed.
```

Comment:
What would be removed in `3.x` -- the whole file?

Reply:
The `$security--theme` variable